airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/AbstractDbSource.java

```diff
@@ -180,6 +183,34 @@ public AutoCloseableIterator<AirbyteMessage> read(final JsonNode config,
         });
   }
 
+  private void validateSourceSchema(Map<String, TableInfo<CommonField<DataType>>> fullyQualifiedTableNameToInfo,
```

Comment:
I think this should be changed to be named `logSourceSchemaChange`

validate implies that if it isn't the same you'd be throwing an error

Reply:
Also add a quick comment on what this function is doing

Reply:
> Also add a quick comment on what this function is doing

renamed this method and added a comment